legacy/dockerregistry/inventory.go

```diff
@@ -1631,6 +1635,9 @@ func MkReadRepositoryCmdReal(
 func MkReadManifestListCmdReal(sc *SyncContext, gmlc *GCRManifestListContext) stream.Producer {
 	var sh stream.HTTP
 
+	// Log GCR query (only if --verbose flag is present)
+	logclient.CountGCRQuery(1)
```

Comment:
Can you just increment a single global variable here (guard it with Mutex lock/unlock)?

Reply:
Same response as above.